README.md

```diff
@@ -1 +1,42 @@
-# basic-express-server
\ No newline at end of file
+# LAB - Class 02
+
+## basic-express-server 
+
+### Author: Matt Ravenmoore
+
+### Links and Resources
+
+- [ci/cd](https://github.com/ravenmoore-401-JS/basic-express-server/actions) (GitHub Actions)
+- [back-end server url](https://rm-basic-server.herokuapp.com/)
+
+- [submit PR](https://github.com/ravenmoore-401-JS/basic-express-server/pull/1)
+
+### Setup
+
+clone down the project from github
+run `npm i`
+
+#### `.env` requirements (where applicable)
+
+- `PORT` - Port Number
+
+#### How to initialize/run your application (where applicable)
+
+run `npm start`
+
+#### How to use your library (where applicable)
+
+#### Tests
+
+- How do you run tests?
+to check tests run `npm test`
+
+- Any tests of note?
+a few basic status tests for server.
+
+- Describe any tests that you did not complete, skipped, etc
+
+#### UML
+
+![UML Diagram](uml.jpg)
+Link to an image of the UML for your application and response to events
```

Comment:
Leaving a note here for general information, I would suggest you have an assets folder with all your images in it just to clean up tree a tiny bit!